### PR TITLE
Fixed label overbound

### LINKED ISSUE #16: Larger word overbounds contentView
###  What did you do?

I used a long word.

### What did you expect to happen?

Truncate the word.

### What happened instead?

When the word is larger than the contentView, it overbounds.
screen shot 2016-11-07 at 00 03 24

### Solution

I've solved it adding this line right after this block [SegmentioCell.swift#L183-L192](https://github.com/Yalantis/Segmentio/blob/34cf755361b490bdea8699bcbd6a4eb267de1e24/Segmentio/Source/Cells/SegmentioCell.swift#L183-L192):

```swift 

segmentTitleLabel.leadingAnchor.constraint(equalTo: containerView.leadingAnchor, constant: 5).isActive = true

```

screen shot 2016-11-07 at 00 03 33
screen shot 2016-11-07 at 00 03 39

### PR DESCRIPTION
**Pull request by LeonardoCardoso**
Close #16

### CODE REVIEW

File: Segmentio/Source/Cells/SegmentioCell.swift [hunk 1]
```diff
@@ -190,6 +190,9 @@ class SegmentioCell: UICollectionViewCell {
                 multiplier: 1,
                 constant: 0
         )
+        if #available(iOS 9, *) {
+            segmentTitleLabel.leadingAnchor.constraint(equalTo: containerView.leadingAnchor, constant: 5).isActive = true
```
Comment (serejahh) [inline Segmentio/Source/Cells/SegmentioCell.swift]: What about iOS 8? Please, rewrite it with NSLayoutConstraint

Comment (LeonardoCardoso) [inline Segmentio/Source/Cells/SegmentioCell.swift]: Sure. It happens on prior iOS 9 devices too.

screen shot 2016-12-31 at 13 10 48
screen shot 2016-12-31 at 13 12 07

**Comment by LeonardoCardoso** [comment]
Now it's working in all iOS versions.

screen shot 2017-01-01 at 01 11 01

**Comment by serejahh** [comment]
Thanks for the PR. I merged it manually

**Pull request closed without merging**